cpp/include/utilities/cython.hpp

```diff
@@ -66,22 +66,22 @@ struct graph_container_t {
     std::unique_ptr<GraphCSCView<int, int, double>> GraphCSCViewDoublePtr;
     std::unique_ptr<GraphCOOView<int, int, float>> GraphCOOViewFloatPtr;
     std::unique_ptr<GraphCOOView<int, int, double>> GraphCOOViewDoublePtr;
-    std::unique_ptr<experimental::graph_view_t<int, int, float, false, false>>
-      graph_view_t_float_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, double, false, false>>
-      graph_view_t_double_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, float, false, true>>
-      graph_view_t_float_mg_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, double, false, true>>
-      graph_view_t_double_mg_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, float, true, false>>
-      graph_view_t_float_transposed_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, double, true, false>>
-      graph_view_t_double_transposed_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, float, true, true>>
-      graph_view_t_float_mg_transposed_ptr;
-    std::unique_ptr<experimental::graph_view_t<int, int, double, true, true>>
-      graph_view_t_double_mg_transposed_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, float, false, false>>
+      graph_t_float_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, double, false, false>>
+      graph_t_double_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, float, false, true>>
+      graph_t_float_mg_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, double, false, true>>
+      graph_t_double_mg_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, float, true, false>>
+      graph_t_float_transposed_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, double, true, false>>
+      graph_t_double_transposed_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, float, true, true>>
+      graph_t_float_mg_transposed_ptr;
+    std::unique_ptr<experimental::graph_t<int, int, double, true, true>>
+      graph_t_double_mg_transposed_ptr;
```

Comment:
So, aren't we adding 64 bit integer support in this release? With only int, we still can't support graphs with more than 2B vertices.

And can't we use int32_t instead of int here? I am not sure about the restrictions but I found this (https://github.com/cython/cython/blob/master/Cython/Includes/libc/stdint.pxd) in googling so I guess it might be possible?

We may need to support (int32_t, int32_t), (int32_t, int64_t), and (int64_t, int64_t) for (vertex_t, edge_t) pairs; and eventually replace int##_t to uint##_t once we retire the code that relies on vertex_t & edge_t being signed integers.

Reply:
yes, I still need to do a pass and address all the 32/64 int sizes issues you bring up once I'm successfully calling Louvain with a `graph_view_t`.

Reply:
@ChuckHastings has some refactorings to the cython factory C++ code which addresses the int64-based feedback, so I'll create another PR now to integrate that and defer these items until then.

Reply:
Confirmed support for int32 and int64 types is present, and I added a `FIXME` for updating to unsigned int types when vertex_t and edge_t code can support unsigned.